opencti-platform/opencti-graphql/src/modules/ingestion/ingestion-csv.graphql

```diff
@@ -23,6 +23,7 @@ type IngestionCsv implements InternalObject & BasicObject {
     ingestion_running: Boolean
     current_state_hash: String
     current_state_date: DateTime
+    markings: [String]
```

Comment:
Can we have null values in this list ? Otherwise I suggest to set the type to `[String!]` which will make sure that the list returns non null values (which is easier to handle afterwards).